src/MassiveActionHandler.ts

```diff
@@ -25,6 +25,7 @@ import { MigrationRunner } from './MigrationRunner'
 export class MassiveActionHandler extends AbstractActionHandler {
   protected allMigrations: Migration[] = []
   protected migrationSequenceByName: { [key: string]: MigrationSequence } = {}
+  protected cyanAuditStatus: boolean | null = null
```

Comment:
Rather than this being nullable, can we default it to false? Since `!cyanAuditStatus` will return the same if null or if false